recipe/meta.yaml

```diff
@@ -13,9 +13,8 @@ source:
 
 build:
   number: 0
-  skip: True  # [py>3]
+  skip: True  # [not py2k]
   script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Doesn't look like `setuptools` is used to build the package. Please drop it from `requirements/build` and please cut this line down to `python setup.py install`.